### PR TITLE
Upgrade build tools to fix deployment on new builds

### PR DESCRIPTION
**Pull request by davkean**
On newer builds of VS, our repo is failing to deploy to VS with:

VSSDK1043: There was a problem finding the extension with a VSIX identifier of "F6B5EACA-7FA1-4591-8D40-A38234763621". Value does not fall within the expected range.

Upgrading our build tools version to a build that contains this fix.

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/project-system

File: src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj [hunk 1]
```diff
@@ -30,7 +30,6 @@
     <ProjectSystemLayer>VisualStudio</ProjectSystemLayer>
     <ProducingSignedVsix>true</ProducingSignedVsix>
     <IsProductComponent>true</IsProductComponent>
-    <InstallRoot>Default</InstallRoot>
```
Comment (srivatsn) [inline src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj]: What does this do?

Comment (davkean) [inline src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj]: Ah, see the second commit.

Comment (srivatsn) [inline src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj]: Doesn't explain what it does? Are we changing behavior of the VSIX somehow?

Comment (davkean) [inline src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj]: Ah, my commit was a little misleading, by "default setting" I mean we setting InstallRoot to it's default value, which happens to be "Default". According to Alex, InstallRoot set to Default *is the* default.

Comment (davkean) [inline src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj]: He's filing/filed a bug to fix this error message.

Comment (srivatsn) [inline src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj]: Ok

**Pull request merged**